open-metadata-implementation/access-services/data-manager/data-manager-spring/src/main/java/org/odpi/openmetadata/accessservices/datamanager/server/spring/DisplayApplicationResource.java

```diff
@@ -836,7 +836,7 @@ public QueryResponse getQueryByGUID(@PathVariable String serverName,
      * UserNotAuthorizedException the user is not authorized to issue this request or
      * PropertyServerException    there is a problem reported in the open metadata server(s)
      */
-    @PostMapping(path = "/elements/{parentGUID}/data-containers")
+    @PostMapping(path = "/schemas/elements/{parentGUID}/data-containers")
```

Comment:
this does not seem right , the comments say it is a form report or query but the end point now references schema which is different from form, report and query.